### ISSUE TITLE
blog: Add entry to KryptonChicken blog posts about Soundszcape

### ISSUE THREAD

**Issue by tommyip** [KryptonChicken/soundzcape#11]
Just document our:
- what are we doing
- current progress
- ideas
- tech stack
- plans

And get more contributors!

**Comment by anshuman73** [comment]
I think this should be done once we have a MVP, that way we'll even get a reference for our website.

**Comment by tommyip** [comment]
Yeah but that will probably take a long time. I personally think that we could just describe what we are currently doing and try to get more people involved 😄 .

**Comment by anshuman73** [comment]
Ahh, yes, getting contributors this way could be useful

**Comment by tommyip** [comment]
Yeah we need contributors with different expertise, i can't be full-full stack developer...
